nsv13/code/modules/overmap/weapons.dm

```diff
@@ -253,6 +250,8 @@
 /obj/structure/overmap/proc/fire_torpedo(atom/target)
 	if(!linked_areas.len && !main_overmap) //AI ships don't have interiors
 		if(torpedoes <= 0)
+			if(ai_controlled)
+				addtimer(VARSET_CALLBACK(src, torpedoes, initial(src.torpedoes)), 60 SECONDS)
```

Comment:
A hacky way to fix this. You may want to rework the entire code with all of this later on so its more modular and easy to use/expand.